apps/dashboard/lib/trpc/routers/api/updateIpWhitelist.ts

```diff
@@ -43,14 +43,27 @@ export const updateApiIpWhitelist = rateLimitedProcedure(ratelimit.update)
             "We are unable to update the API whitelist. Please try again or contact support@unkey.dev",
         });
       });
-    if (!api || api.workspace.tenantId !== ctx.tenant.id) {
+
+    if (
+      !api ||
+      api.workspace.tenantId !== ctx.tenant.id ||
+      input.workspaceId !== api.workspace.id
+    ) {
       throw new TRPCError({
         code: "NOT_FOUND",
         message:
           "We are unable to find the correct API. Please try again or contact support@unkey.dev.",
       });
     }
 
+    if (api.workspace.plan !== "enterprise") {
```

Comment:
actually this is a flag on the workspace.features
we decoupled features from plans, cause it's sometimes nice to give someone a feature regardless of their plan

Reply:
Yep makes sense will adjust api and frontend to take that into account